vector/src/main/res/values/strings.xml

```diff
@@ -3698,11 +3699,22 @@
     <string name="delete_poll_dialog_title">Remove poll</string>
     <string name="delete_poll_dialog_content">Are you sure you want to remove this poll? You won\'t be able to recover it once removed.</string>
 
+    <!-- Location -->
+    <string name="location_activity_title_static_sharing">Share location</string>
+    <string name="location_activity_title_preview">Location</string>
+    <string name="a11y_location_share_icon">Share location</string>
+    <string name="location_share">Share location</string>
+    <string name="location_not_available_dialog_title">Element could not access your location</string>
```

Comment:
Please use `template_` string and not hard-coded `Element` name. See other example in this file.

Reply:
Of course. Done.